Exonoesis/Helping-Hand
# Update to Bevy version 0.10

## What is the purpose of these changes?
To update Bevy and the other project dependencies to their most current versions.

## What has changed?
Bevy is now version 0.10, Bevy ECS LDtk is version 0.6, and Bevy Kira Audio is version 0.15. Also updated code that needed to be altered due to migration.

## How can I verify this works?
1. Check out branch
2. Run Unit Tests and observe that they all pass
3. Launch game and visually verify that it loads and functions as expected
